.github/workflows/github-action-trigger-downstream-job.yaml

```diff
@@ -0,0 +1,49 @@
+name: Github Action to trigger downstream job
+on:
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  jobs01:
+    # Check if comment contains /trigger github-experimental
+    if: contains(github.event.comment.body, '/trigger github-experimental')
+    runs-on: [ubuntu-latest]
+    permissions:
+      pull-requests: write
+    outputs:
+      approvers_check: ${{ steps.approvers_check.outputs.approvers_check }}
+    steps:
+      - name: Install wget and tar
+        run: sudo apt-get install wget tar
+
+      - name: Download yq binary
```

Comment:
isn't there already some pre-built image with yq binary on the marketplace? That would allow to NOT have to manage that dependency over new releases, security and so on...

Reply:
https://github.com/mikefarah/yq?tab=readme-ov-file#github-action